behat/features/installation.feature

```diff
@@ -0,0 +1,10 @@
+@api
+Feature: Installation
```

Comment:
This feature in the behat quick starter is useful for allowing the user to see some output to know they've set it up properly.